tools/please_pex/pex/pex.go

```diff
@@ -161,10 +156,22 @@ func (pw *Writer) Write(out, moduleDir string) error {
 	f := zip.NewFile(out, true)
 	defer f.Close()
 
-	// Write preamble (i.e. the shebang that makes it executable)
-	if err := f.WritePreamble([]byte(pw.shebang)); err != nil {
+	// Write preamble (i.e. the binary that makes the .pex executable)
+	nf := mustOpen("preamble")
```

Comment:
`nf`?

Reply:
I think the thought process was "a **f**ile representing the **n**ative-code preamble". Intended to mirror the `f` above it, but a poor choice in retrospect.